Meadowlark-js/backends/meadowlark-mongodb-backend/package.json

```diff
@@ -19,9 +19,9 @@
     "build:copy-non-ts": "copyfiles -u 1 -e \"**/*.ts\" \"src/**/*\" dist --verbose"
   },
   "dependencies": {
-    "@edfi/meadowlark-authz-server": "0.3.0-dev.0",
-    "@edfi/meadowlark-core": "0.3.0-dev.0",
-    "@edfi/meadowlark-utilities": "0.3.0-dev.0",
+    "@edfi/meadowlark-authz-server": "^v0.3.0-pre-35",
+    "@edfi/meadowlark-core": "^v0.3.0-pre-35",
+    "@edfi/meadowlark-utilities": "^v0.3.0-pre-35",
```

Comment:
In all of my excitement to see if I have this process working, without committing changes to the package.json every time the version updates, I forgot to test and make sure this is actually working. The key question: if I make an update to the meadowlark-utilities, will the other packages pickup that change? Or are they going to try to pull pre-35 from Azure Artifacts and use it instead?